testing/trino-product-tests-launcher/src/main/resources/docker/presto-product-tests/conf/environment/multinode-iceberg-minio-cached/iceberg.properties

```diff
@@ -3,8 +3,8 @@ hive.metastore.uri=thrift://hadoop-master:9083
 fs.cache.enabled=true
 fs.cache.directories=/tmp/cache/iceberg
 fs.cache.max-disk-usage-percentages=90
-hive.s3.aws-access-key=minio-access-key
-hive.s3.aws-secret-key=minio-secret-key
-hive.s3.endpoint=http://minio:9080/
-hive.s3.path-style-access=true
-hive.s3.ssl.enabled=false
+fs.native-s3.enabled=true
```

Comment:
Can we also disable the legacy libs with `fs.hadoop.enabled=false`?

Reply:
ya this will require slightly more changes.. I will update this PR with the required changes. Looks like we can apply the same for many other product-tests which were already migrated.

Reply:
updated